apps/mobile-alphabet/src/app/Components/alphabet-card-detail.presenter.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Text, View } from 'react-native';
+import config from './Config.json';
+import { AlphabetData } from './Menu';
+
+export function AlphabetCardDetailPresenter() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    // Sequence numbers are indexed starting at 1
+    const [selectedLetterSequenceNumber, setSelectedLetterSequenceNumber] = useState<number>(1);
+
+    useEffect(() => {
+        //TODO cache this after fetching it once
+        const fetchData = async () => {
+            try {
+                //TODO use context api
+                const response = await fetch(config.apiUrl, {
+                    mode: 'cors',
+                });
+                setAlphabetData(await response.json());
+            } catch (error) {
+                console.error('Error fetching data:', error);
+            }
+        };
+
+        fetchData();
+    }, []);
+
+    if (!alphabetData) {
+        return <Text>Loading...</Text>;
+    }
+
+    const {
+        data: { alphabet_cards: alphabetCards },
+    } = alphabetData;
+
+    const selectedCard = alphabetCards.find(({ sequence_number: sequenceNumber }) => {
+        return Number.parseInt(sequenceNumber) === selectedLetterSequenceNumber;
+    });
+
+    const {
+        word,
+        letter,
+        sequence_number,
+        card_image,
+        letter_audio,
+        word_audio,
+        standalone_image,
+    } = selectedCard;
+    return (
```

Comment:
Put everything up to this line back in the original parent component, which should at this point read:
```tsx
return (
    <AlphabetCardPresenter {...selectedCard} />
)
where I have used the spread syntax to simply flow all `selected card` props into `AlphabetCardPresenter` as its props. 
```

Reply:
I see that you've done this now. Good work.